src/main/java/org/broadinstitute/hellbender/tools/exome/segmentation/PerformAlleleFractionSegmentation.java

```diff
@@ -18,7 +19,7 @@
  * Groups contiguous targets with the same minor allele fraction for a single sample.
  *
  *  <p>
- *     The --tumorHets file is from {@link GetBayesianHetCoverage}, preferably, but can also come from {@link org.broadinstitute.hellbender.tools.exome.GetHetCoverage}.
+ *     The --tumorHets file is from {@link org.broadinstitute.hellbender.tools.exome.GetBayesianHetCoverage}, preferably, but can also come from {@link org.broadinstitute.hellbender.tools.exome.GetHetCoverage}.
```

Comment:
Thanks for this! I looked and it appears I am already following these preferences:

<img width="1032" alt="screenshot 2017-06-05 23 43 32" src="https://cloud.githubusercontent.com/assets/11543866/26812945/deb08b32-4a48-11e7-9658-9c337ce1a9cd.png">

For some links, I don't get this appendage. For other links, I get the red font and then so I use the appendage. Not clear to me why there is a distinction between tools.